mmv1/products/dlp/DeidentifyTemplate.yaml

```diff
@@ -68,18 +73,110 @@ properties:
+          - deidentify_config.0.image_transformations
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: 'transforms'
+            required: true
+            description: |
+              For determination of how redaction of images should occur.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'redactionColor'
+                  description: |
+                    The color to use when redacting content from an image. If not specified, the default is black.
+                  properties:
+                    - !ruby/object:Api::Type::Double
+                      name: red
+                      description: |
+                        The amount of red in the color as a value in the interval [0, 1].
+                      default_value: 0.0
+                    - !ruby/object:Api::Type::Double
+                      name: blue
+                      description: |
+                        The amount of blue in the color as a value in the interval [0, 1].
+                      default_value: 0.0
+                    - !ruby/object:Api::Type::Double
+                      name: green
+                      description: |
+                        The amount of green in the color as a value in the interval [0, 1].
+                      default_value: 0.0
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'selectedInfoTypes'
+                  description: Apply transformation to the selected infoTypes.
+                  exactly_one_of:
+                    - transforms.0.selected_info_types
+                    - transforms.0.all_info_types
+                    - transforms.0.all_text
+                  properties:
+                  - !ruby/object:Api::Type::Array
+                    name: 'infoTypes'
+                    description: |
+                      InfoTypes to apply the transformation to. Leaving this empty will apply the transformation to apply to
+                      all findings that correspond to infoTypes that were requested in InspectConfig.
+                    item_type: !ruby/object:Api::Type::NestedObject
+                      properties:
+                        - !ruby/object:Api::Type::String
+                          name: 'name'
+                          required: true
+                          description: |
+                            Name of the information type.
+                        - !ruby/object:Api::Type::String
+                          name: 'version'
+                          description: |
+                            Version name for this InfoType.
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'allInfoTypes'
+                  description: Apply transformation to all findings not specified in other ImageTransformation's selectedInfoTypes.
+                  exactly_one_of:
```

Comment:
`exactly_one_of` can be removed.

Reply:
Removed `exactlyOneOf` from all the three fields.